i18n/update_translations.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
I don't want to put this in a Java application. It's just a simple helper script and it was easy to implement in python (this is nothing which would usually be done in JS). I don't see a problem with a different programming language because you just have to use it (i.e. there's no need to learn python to translate your scripts).